src/client/results/Results.js

```diff
@@ -6,7 +6,8 @@ import MostActiveUsers from './mostfrequent/users/MostActiveUsers';
 import { groupedCountWords, mostFrequentWords, mostFrequentUsers } from './../tweetAnalysis';
 
 let Results = ({ feed, mostFrequent }) => {
-  if (feed.length === 0) {
+  const posts = feed.posts;
+  if (posts.length === 0) {
```

Comment:
how comes not just changing the results paramater to posts from feed ? guessing this is just a clarity thing